examples/TooltipExample.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright Schrodinger, LLC
+ */
+
+"use strict";
+
+const FakeObjectDataListStore = require('./helpers/FakeObjectDataListStore');
+const { ImageCell, TooltipCell } = require('./helpers/cells');
+const { Table, Column, Cell } = require('fixed-data-table-2');
+const React = require('react');
+const ReactTooltip = require('react-tooltip');
+const {findDOMNode} = require('react-dom');
+
+class TooltipExample extends React.Component {
```

Comment:
Example to assist questions like https://github.com/schrodinger/fixed-data-table-2/issues/48